projects/fsastorefrontlib/src/assets/translations/de/b2b.de.ts

```diff
@@ -15,9 +15,13 @@ export const organization = {
     header: '[DE] All organizations ({{count}})',
     unit: '[DE] Organization',
     parentUnit: '[DE] Parent Organization',
+    hint:
+      '[DE] Organizations represent departments, stores, regions, or any other logical grouping that makes sense to you. Members have access to all child organizations of their primary organization.',
     details: {
       title: '[DE] Organization Details',
       subtitle: '[DE] Organization: {{ item.name }}',
+      hint:
+        '[DE] Organizations represent departments, stores, regions, or any other logical grouping that makes sense to you. Disabling a organization disables all children of the organization, including child organizations and members. Disabled organizations cannot be edited.',
```

Comment:
`Disabling a organization` should be `Disabling an organization`? Please check with @marija-ristin-msg